pillar/heroku/xpro.sls

```diff
@@ -147,20 +148,20 @@ heroku:
     MITXPRO_BASE_URL: {{ env_data.MITXPRO_BASE_URL }}
     MITXPRO_DB_CONN_MAX_AGE: 0
     MITXPRO_DB_DISABLE_SSL: True    # pgbouncer buildpack uses stunnel to handle encryption
-    MITXPRO_EMAIL_HOST: __vault__::secret-operations/global/mit-smtp>data>relay_host
-    MITXPRO_EMAIL_PASSWORD: __vault__::secret-operations/global/mit-smtp>data>relay_password
-    MITXPRO_EMAIL_PORT: 587
+    MITXPRO_EMAIL_HOST: {{ smtp_config.relay_host }}
+    MITXPRO_EMAIL_PASSWORD: {{ smtp_config.relay_password }}
+    MITXPRO_EMAIL_PORT: {{ smtp_config.relay_port }}
     MITXPRO_EMAIL_TLS: True
-    MITXPRO_EMAIL_USER: __vault__::secret-operations/global/mit-smtp>data>relay_username
+    MITXPRO_EMAIL_USER: {{ smtp_config.relay_username }}
     MITXPRO_ENVIRONMENT: {{ env_data.env_name }}
-    MITXPRO_FROM_EMAIL: 'MIT xPRO <xpro@mit.edu>'
+    MITXPRO_FROM_EMAIL: 'MIT xPRO <xpro@xpro.mit.edu>'
     MITXPRO_LOG_LEVEL: {{ env_data.app_log_level }}
     MITXPRO_OAUTH_PROVIDER: 'mitxpro-oauth2'
     MITXPRO_REGISTRATION_ACCESS_TOKEN:  __vault__:gen_if_missing:secret-{{ business_unit }}/{{ env_data.openedx_environment }}/xpro-registration-access-token>data>value
-    MITXPRO_REPLY_TO_ADDRESS: 'MIT xPRO <xpro@mit.edu>'
+    MITXPRO_REPLY_TO_ADDRESS: 'MIT xPRO <xpro@xpro.mit.edu>'
```

Comment:
I'll have to check on this as well. I assume this will route to zendesk, but I'm checking.

Reply:
I don't think I'm going to get answer today, and changing this again seems like it would be relatively minor. 

Let's go with: 

```suggestion
    MITXPRO_REPLY_TO_ADDRESS: 'MIT xPRO <support@xpro.mit.edu>'
```